_shows/17_18/death_of_the_author.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Death of the Author
+devised: True
+period: Autumn
+student_written: true
+season: Fringe
+season_sort: 140
+date_start: 2017-12-10
+date_end: 2017-12-11
+venue: New Theatre
+
+cast:
+- role: Dorian
+  name: Francis Simmons
+- role: John Watson
+  name: Charlie Basley
+- role: Lady Macbeth
+  name: Beth Carter
+- role: The Jabberwocky
+  name: Emily Wong
+- role: Lizzy
+  name: Eleanor Rickenbach
+
+crew:
+- role: Director
+  name: Daniel McVey
+- role: Producer
+  name: Florence Bell
+- role: Technical Director
+  name: Hannah McCullough
+- role: Production Assistant
+  name: Zoe Smith
+- role: Design Assistant
+  name: Laura Wolczyk
+- role: Lighting Designer
+  name: Hannah Burne
```

Comment:
Can I be added as lighting programmer please?

Reply:
I guess